website/docs/networking/vpc-lattice/latticeabtesting.md

```diff
@@ -0,0 +1,119 @@
+# Set up Lattice Service Network
+
+Create the Kubernetes Gateway `eks-workshop-gw`:
+   ```bash
+   $ kubectl apply -f workspace/modules/networking/vpc-lattice/controller/eks-workshop-gw.yaml
+   ```
+Verify that `eks-workshop-gw` gateway is created:
+   ```bash
+   $ kubectl get gateway  
+   ```
+   ```
+   $ NAME              CLASS         ADDRESS   READY   AGE
+   $ eks-workshop-gw   aws-lattice                     12min
+   ```
+
+Once the gateway is created, find the VPC Lattice service network. Wait until the status is `Reconciled` (this could take about five minutes).
+   ```bash
+   $ kubectl describe gateway eks-workshop-gw
+   ```
+   ```
+   $ apiVersion: gateway.networking.k8s.io/v1alpha2
+   kind: Gateway
+   ...
+   $ status:
+   conditions:
+   message: 'aws-gateway-arn: arn:aws:vpc-lattice:us-west-2:<YOUR_ACCOUNT>:servicenetwork/sn-03015ffef38fdc005'
+   reason: Reconciled
+   status: "True"
+   ```
+ Now you can see the associated Service Network created in the VPC console under the Lattice resources.
+![Checkout Service Network](assets/servicenetwork.png)
+
+# Create Routes to targets
+We will show how to preform A/B testing between the two versions using `HTTPRoutes`.
+
+At the time of writing (Apr 2023), the controller requires a port number for `targetPort` . We are working on a better solution, progress is tracked [here](https://github.com/aws/aws-application-networking-k8s/issues/86).
+
+```bash
+$ kubectl patch svc checkout -n checkout --patch '{"spec": { "type": "ClusterIP", "ports": [ { "name": "http", "port": 80, "protocol": "TCP", "targetPort": 8080 } ] } }'
+```
+
+Create the Kubernetes `HTTPRoute` route that evenly distributes the traffic between `checkout` and `checkoutv2`:
+   ```bash
+   $ kubectl apply -f workspace/modules/networking/vpc-lattice/routes/checkout-route.yaml
+   ```
+   ```file
+   /networking/vpc-lattice/routes/checkout-route.yaml
+   ```
+
+Find out the `HTTPRoute`'s DNS name from `HTTPRoute` status (highlighted here on the `message` line):
+
+```bash
+$ kubectl describe httproute checkoutroute -n checkout
+```
+
+```bash
+$ apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: HTTPRoute
+    ...
+status:
+parents:
+- conditions:
+    - lastTransitionTime: "2023-02-27T14:36:26Z"
+    message: 'DNS Name: checkout-checkouroute-05bcd5fb087c79394.7d67968.vpc-lattice-svcs.us-west-2.on.aws'
+    reason: Reconciled
+    status: "True"
+    type: httproute
+    controllerName: application-networking.k8s.aws/gateway-api-controller
+    parentRef:
+    group: gateway.networking.k8s.io
+    kind: Gateway
+    name: eks-workshop-gw
+```
+
+ Now you can see the associated Service created in the VPC console under the Lattice resources.
+![CheckoutRoute Service](assets/checkoutroute.png)
+
+Patch the configmap to point to the new endpoint.
+
+```bash
+$ export CHECKOUT_ROUTE_DNS='http://'$(kubectl get httproute checkoutroute -n checkout -o json | jq -r '.status.parents[0].conditions[0].message' | cut  -c 11-)
```

Comment:
is the checkout route static?

Reply:
No, is dynamically generated by Lattice .